imports/plugins/core/ui/client/components/loading/loading.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component, PropTypes } from "react";
+import assign from "domkit/appendVendorPrefix";
+import insertKeyframesRule from "domkit/insertKeyframesRule";
+
+// Loading Animations
+// inspired by http://madscript.com/halogen
+
+class Loading extends Component {
```

Comment:
Is this a 3rd loading/progress indicator?  I think we have two others already... can we standardize/use one?